mockgcp/mockcontainer/testdata/clusters/crud/_http.log

```diff
@@ -218,8 +229,10 @@ X-Xss-Protection: 0
   "name": "test-${uniqueId}",
   "network": "default",
   "networkConfig": {
+    "defaultSnatStatus": {},
     "network": "default",
-    "subnetwork": "default"
+    "serviceExternalIpsConfig": {},
+    "subnetwork": "projects/${projectId}/regions/us-central1/subnetworks/default"
```

Comment:
"default" -> "projects/${projectId}/regions/us-central1/subnetworks/default"

Was the old value bad? If not are we sure that this change is ok?

Reply:
The old value was not "bad", but it was not the format in the response from the real API.